### PR TITLE
RLS: update version to 1.3.0rc2

### PR DESCRIPTION
**Pull request by qwhelan**
Create a conda-forge build of bottleneck 1.3.0rc2 per CFEP-05.

Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: .ci_support/linux_python2.7.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ c_compiler_version:
 channel_sources:
 - conda-forge,defaults
 channel_targets:
-- conda-forge main
+- conda-forge rc
```
Comment (ocefpaf) [inline .ci_support/linux_python2.7.yaml]: I guess that the cfep-05 text suggests `rc_<package-name>`, so this should be `rc_bottleneck`, right? This avoids problems of installing more stuff from other `rc`s labels b/c conda label are "channels."

File: recipe/conda_build_config.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+numpy:     # [aarch64]
+  - '1.16' # [aarch64]
+
+channel_targets:
+  - conda-forge rc
```
Comment (ocefpaf) [inline recipe/conda_build_config.yaml]: All you need is to change the name here and rerender.
We also need to create an `rc` branch for it.

File: .ci_support/linux_python2.7.yaml (hunk 1 above)
Comment (qwhelan) [reply .ci_support/linux_python2.7.yaml]: Thanks, the discussion around the format here made it a bit unclear what the correct value was. Updated to your suggestion.

File: recipe/conda_build_config.yaml (hunk 2 above)
Comment (qwhelan) [reply recipe/conda_build_config.yaml]: Yeah, happy to submit this as a PR to a `rc` branch - just can't create it myself, apparently.

File: recipe/LICENSE.txt [hunk 3]
```diff
@@ -0,0 +1,13 @@
+BSD 3-clause license
```
Comment (jakirkham) [inline recipe/LICENSE.txt]: It looks like a re-rendering occurred in the `recipe` directory and added this stuff there, which shouldn't be there. Could you please clean out the `recipe` directory of everything except `meta.yaml` and `conda_build_config.yaml`?

Comment (qwhelan) [reply recipe/LICENSE.txt]: Done

File: recipe/.gitignore [hunk 4]
```diff
@@ -0,0 +1,3 @@
+*.pyc
```
Comment (jakirkham) [inline recipe/.gitignore]: Sorry also these hidden files should be removed to please.

Comment (qwhelan) [reply recipe/.gitignore]: Just removed those as well

**Comment by qwhelan** [comment]
Closing as the UI for creating a new branch just showed up. Will reopen targeting `rc`

File: recipe/conda_build_config.yaml (hunk 2 above)
Comment (qwhelan) [reply recipe/conda_build_config.yaml]: Implemented as #25

**Pull request closed without merging**